Tests/Testo/Invoice/SalesOrderCustom/SalesOrderCustomServiceTest.php

```
@@ -0,0 +1,122 @@
+use Testo\Test;
+
+/**
+ * Covers SalesOrderCustomService: saveSoCustom's sales-order/custom-field
+ * relation persistence and field assignment, and deleteSalesOrderCustom.
+ */
+#[Test]
+final class SalesOrderCustomServiceTest
+{
+    private function makeService(
+        ?SalesOrderCustomRepository $repository = null,
+        ?CustomFieldRepository $customFieldRepository = null,
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $customFieldRepository. Keep variable name length under 20. (undefined)

(LongVariable)